src/test/oauth.spec.js

```diff
@@ -0,0 +1,73 @@
+import chai from 'chai';
+import nock from 'nock';
+import app from '../index';
+
+const { expect } = chai;
+
+describe('SOCIAL LOGIN', () => {
+  it('should hit the facebook URL for social login', async () => {
+    nock('https://www.facebook.com/')
+      .filteringPath(() => '/auth/facebook')
+      .get('/auth/facebook')
+      .reply(200, { message: 'hit the route' });
+
+    const res = await chai.request(app).get('/auth/facebook');
+    expect(res.body.message).to.include('hit the route');
```

Comment:
why are we sending this message to users? 🤷‍♂ 

I don't think it provides any business value